src/api/v4/filter/bounding-box-leaflet.js

```diff
@@ -53,4 +57,9 @@ BoundingBoxLeaflet.prototype.$getInternalModel = function () {
   return this._internalModel;
 };
 
+// Helper to check if an element is a leafletmap object
+function _isLeafletMap (element) {
+  return element instanceof L.Map;
```

Comment:
Same check as with google here. We're relying in a global object that can be not present.